specification/search/data-plane/Azure.Search/preview/2020-06-30-Preview/searchindex.json

```diff
@@ -245,6 +251,93 @@
+            "in": "query",
+            "type": "string",
+            "enum": [
+              "none",
+              "en-us"
+            ],
+            "x-ms-enum": {
+              "name": "QueryLanguage",
+              "modelAsString": true,
+              "values": [
+                {
+                  "value": "none",
+                  "name": "None",
+                  "description": "Query language not specified."
+                },
+                {
+                  "value": "en-us",
+                  "name": "EnUs",
+                  "description": "English"
+                }
+              ]
+            },
+            "description": "The language of the query.",
+            "x-ms-parameter-grouping": {
+              "name": "SearchOptions"
+            }
+          },
+          {
+            "name": "speller",
+            "in": "query",
+            "type": "string",
+            "enum": [
+              "none",
+              "lexicon"
+            ],
+            "x-ms-enum": {
+              "name": "Speller",
+              "modelAsString": true,
+              "values": [
+                {
+                  "value": "none",
+                  "name": "None",
+                  "description": "Speller not enabled."
+                },
+                {
+                  "value": "lexicon",
+                  "name": "Lexicon",
+                  "description": "Speller corrects individual query terms using a static lexicon for the language specified by the queryLanguage parameter."
+                }
+              ]
+            },
+            "description": "Improve search recall by spell-correcting individual search query terms.",
+            "x-ms-parameter-grouping": {
+              "name": "SearchOptions"
+            }
+          },
+          {
+            "name": "answers",
+            "in": "query",
+            "type": "string",
+            "enum": [
+              "none",
+              "extractive"
+            ],
+            "x-ms-enum": {
+              "name": "Answers",
+              "modelAsString": true,
+              "values": [
+                {
+                  "value": "none",
+                  "name": "None",
+                  "description": "Do not return answers for the query."
+                },
+                {
+                  "value": "extractive",
+                  "name": "Extractive",
+                  "description": "Extracts answer candidates from the contents of the documents returned in response to a query expressed as a question in natural language."
+                }
+              ]
+            },
+            "description": "This parameter is only valid if the query type is 'semantic'. If set, the query returns answers extracted from key passages in the highest ranked documents. The number of answers returned can be configured by appending the |count-<number of answers> option after the answers parameter value. Default is 1.",
```

Comment:
What the pipe "|" character for before "count"?

Reply:
This is the syntax Azure Search currently use to add "sub" parameters at query time.  An example input would be
answers=extractive|count-1

We already use a similar syntax for highlights:
highlight=title|count-5,description|count-3

Reply:
Thanks for explaining. I would suggest rewording to make that clearer for customers who may not know. I help develop the Search SDK and wasn't aware. 🙂